doc/api/tracing.md

```diff
@@ -22,6 +22,8 @@ The available categories are:
 * `node.bootstrap`: Enables capture of Node.js bootstrap milestones.
 * `node.console`: Enables capture of `console.time()` and `console.count()`
   output.
+* `node.threadpool`: Enables capture of Node.js threadpool,
+  except asynchronous file IO and DNS.
```

Comment:
I'd find it would be more helpful to document what's included, rather than what's not included.

Reply:
Done !